client/reducers/postsReducer.js

```diff
@@ -2,11 +2,11 @@ import * as types from '../constants/actionTypes';
 
 const initialState = {
   title: 'Noise Disturbance',
-  location: 'Codesmith HQ',
+  steet_name: 'Codesmith HQ',
   details: 'Reports of excessive hollering and clapping noise',
   time: '6:30pm',
-  imgUrl: 'https://static.frontendmasters.com/assets/teachers/sentance/thumb@2x.jpg',
-  videoUrl: 'https://www.youtube.com/embed/sqAwvpw_FKc',
+  image_url: 'https://static.frontendmasters.com/assets/teachers/sentance/thumb@2x.jpg',
+  video_url: 'https://www.youtube.com/embed/sqAwvpw_FKc',
   expandedPost: false,
```

Comment:
Should we change initialState to null values now that we have the map Component working?

Reply:
no, the 'post' on the map is always visible, so it needs a default value

Reply:
Good point!
just set the post initial state to null and let post not show up the moment page renders. 
pushing again now

Reply:
i got it disappear if the title is a null value. depends on how you guys want the app to be.

Reply:
> no, the 'post' on the map is always visible, so it needs a default value

i didn't refresh page when i reply to huy. didn't see ruzeb's post sry.

Reply:
it's actually fine, i thought it would break with nulls but i tested it out and it didnt